portfolio/src/main/webapp/index.html

```diff
@@ -35,7 +38,41 @@ <h5 class="mt-0">Projects & Experience</h5>
         <li class="list-group-item">Hackathon Residencias ITESM MTY 2019</li>
         <li class="list-group-item">Mock interview group</li>
         <li class="list-group-item">International Collegiate Programming Contest</li>
+        <li class="list-group-item">Banorte Security Project Management System</li>
       </ul>
+      <h5 class="mt-3">Gallery</h5>
+      <table class="table table-borderless text-center">
+        <tbody>
+          <tr>
+            <td>
+              <img src="./images/gallery/sterling-ruby.jpg" alt="Sterling Ruby QUILT" class="gallery-image">
+            </td>
+            <td>
+              <img src="./images/gallery/cars.jpg" alt="Chris LaBrooy artwork" class="gallery-image">
+            </td>
+            <td>
+              <img src="./images/gallery/pleasures.jpg" alt="Pleasures Store" class="gallery-image">
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <img src="./images/gallery/cdg.jpg" alt="CDG x Nike Dunk Low" class="gallery-image">
+            </td>
+            <td>
+              <img src="./images/gallery/yeezus.jpg" alt="Distorted Yeezus cover" class="gallery-image">
+            </td>
+            <td>
+              <img src="./images/gallery/fdmtl.jpg" alt="FDMTL patchwork tee" class="gallery-image">
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <br>
+      <br>
+      <br>
```

Comment:
Since the image gallery and sendToRandomVideo button are two very different features, I think it would be better to  wrap them in classes and then use css to create the spacing between them, rather than using multiple <br> statements.

Reply:
Done! Changed it with bootstrap styling